DigitalOffice/Services/TimeManagementService/Validators/ChangeWorkTimeRequestValidator.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.TimeManagementService.Models;
+using System;
+
+namespace LT.DigitalOffice.TimeManagementService.Validators
+{
+    public class ChangeWorkTimeRequestValidator : AbstractValidator<ChangeWorkTimeRequest>
+    {
+        public ChangeWorkTimeRequestValidator()
```

Comment:
I think, this validator shoul be like CreateTimeValidator. Need more validation rules

Reply:
fixed